Formula/f/ftxui.rb

```diff
@@ -0,0 +1,42 @@
+class Ftxui < Formula
+  desc "C++ Functional Terminal User Interface"
+  homepage "https://github.com/ArthurSonzogni/FTXUI"
+  url "https://github.com/ArthurSonzogni/FTXUI/archive/refs/tags/v6.0.2.tar.gz"
+  sha256 "ace3477a8dd7cdb911dbc75e7b43cdcc9cf1d4a3cc3fb41168ecc31c06626cb9"
+  license "MIT"
+  head "https://github.com/ArthurSonzogni/FTXUI.git", branch: "main"
+
+  depends_on "cmake" => :build
+
+  def install
+    args = std_cmake_args + %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DFTXUI_BUILD_DOCS=ON
+      -DFTXUI_BUILD_EXAMPLES=OFF
+      -DFTXUI_BUILD_TESTS=OFF
+      -DFTXUI_QUIET=ON
+      -DFTXUI_ENABLE_COVERAGE=ON
+    ]
+
+    system "cmake", "-S", ".", "-B", "builddir", *args
```

Comment:
```suggestion
    args = %W[
      -DBUILD_SHARED_LIBS=ON
      -DCMAKE_INSTALL_RPATH=#{rpath}
      -DFTXUI_BUILD_DOCS=ON
      -DFTXUI_BUILD_EXAMPLES=OFF
      -DFTXUI_BUILD_TESTS=OFF
      -DFTXUI_QUIET=ON
      -DFTXUI_ENABLE_COVERAGE=ON
    ]

    system "cmake", "-S", ".", "-B", "builddir", *args, *std_cmake_args
```
We need `CMAKE_INSTALL_RPATH` since some libraries are linked with `@rpath` references but are not linked with an `LC_RPATH` command.

Also, we always pass `std_cmake_args` last.

Why do we need `FTXUI_ENABLE_COVERAGE=ON`?

Reply:
the COVERAGE is optional and can be removed